azure-monitor-opentelemetry-distro/README.md

```diff
@@ -28,6 +28,18 @@ Install the Azure Monitor Opentelemetry Distro with [pip][pip]:
 pip install azure-monitor-opentelemetry-distro --pre
 ```
 
+### Manual Configuration
+
+You can use the `configure_opentelemetry` to set up manual instrumentation in a single line and send telemetry for your app to Azure Monitor. `configure_opentelemetry` supports the following optional arguments:
+
+* connection_string - You can pass in your connections string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 
+* service_name = Specefies the service name. 
+* service_namespace = Specefies the service namespace.
+* service_instance_id = Specefies the instance id.
+* disable_tracing = If set to `True`, disables collection and export or trace telemetry
+* logging_level = Determines the logging level of the Opentelemetry Logging Handler. Ex: logging.WARNING
+* disable_logging = If set to `True`, disables collection and export or trace telemetry
+
 ### Prerequisites:
 To use this package, you must have:
 * Azure subscription - [Create a free account][azure_sub]
```

Comment:
Nit: `application_insights_namespace` link is broken. Replace with [this](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)

Reply:
Done